cloud/src/autogluon/cloud/predictor/predictor.py

```diff
@@ -897,3 +951,125 @@ def _get_local_predictor_cls(self):
         from autogluon.text import TextPredictor
         predictor_cls = TextPredictor
         return predictor_cls
+
+
+class ImageCloudPredictor(CloudPredictor):
```

Comment:
More broad question but do we want to call this ImageSMCloudPredictor and differentiate between logic that is SM specific and generic? Doesn't need to be in this PR, but something to think about in-case we want to add support for solutions other than SM.

Reply:
I think maybe letting the user to specify a backend is better? This way they don't need to mess around with all different kinds of CloudPredictor class. Open to discussion too

Reply:
Ok, that sounds fine for now.